src/amrutaM/testcases/TC3AlertUsingJavaScript.java

```diff
@@ -0,0 +1,48 @@
+package amrutaM.testcases;
+
+/*TC#3 - click Alert button
+Verify - "You must be TechnoCredits student"
+*/
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import amrutaM.base.PredefinedActions;
+
+public class TC3AlertUsingJavaScript {
+	static WebDriver driver;
```

Comment:
you have create instance but it's value is null as you have not pass value it local variable to static variable